stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/plugin/factory/StreamOfflinePluginFactory.kt

```diff
@@ -76,8 +117,9 @@ public class StreamOfflinePluginFactory(
             deleteReactionListener = DeleteReactionListenerImpl(
                 logic = logic,
                 globalState = globalStateRegistry,
-                repos = chatDomainImpl.repos,
+                repos = repos,
             ),
+            stateHandler = stateHandler
```

Comment:
Why do we have to pass this stateHandler to OfflinePlugin?

Reply:
Not really, taking a look, StateHandler is not really necessary